### PR TITLE
fix(PDFViewer): Add support for non-latin characters

### PR DESCRIPTION
**Pull request by gustavlrsn**
Resolve https://github.com/opencollective/opencollective/issues/6822

# Description

Adds support for non-latin characters by [providing cmaps to `react-pdf`](https://github.com/wojtekmaj/react-pdf#support-for-non-latin-characters)

# Outstanding issue

- When adding the cmaps option to `react-pdf`s `<Document />` the rendering behavior seems to change:
  - it's adding a loading step for the individual pages (perhaps expected to render the page with the cmaps?) 
  - an error is thrown "Transport destroyed", screenshot below
  - the error is quickly removed and after which the pdf page seems to render fine

image

- [x] Issue in created in `react-pdf`: https://github.com/wojtekmaj/react-pdf/issues/1533
- [x] A potential quick fix is included in this PR to maintain existing behavior by setting `loading` and `error` on `<Page/>` to render as `null`

### CODE REVIEW

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 15  | ❗       |
    


<details>
<summary>❗ storybook/collective-page-hero--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/design-system-collective-navbar--customize-ct-as (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/design-system-collective-navbar--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/design-system-styled-collective-card--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/design-system-styled-collective-card--frozen-collective (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-form--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--invoice (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--loading (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--receipt (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expenses-list--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expenses-list--states (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/host-dashboard-unhost-account-modal--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/host-dashboard-unhost-account-modal--with-balance (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/o-auth-application-approve-screen--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/virtual-cards-edit-virtual-card-modal--edit (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

File: public/static/cmaps/.keep
Comment (Betree) [inline public/static/cmaps/.keep:1]: Why do we need to keep this folder? `CopyPlugin` is supposed to create the full path, isn't it?

**Review by Betree — APPROVED** [review]

File: public/static/cmaps/.keep
Comment (gustavlrsn) [reply public/static/cmaps/.keep:1]: Maybe it does, did not consider that, I'll try!

Comment (gustavlrsn) [reply public/static/cmaps/.keep:1]: It does, removed the .keep file

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 15  | ❗       |
    


<details>
<summary>❗ storybook/collective-page-hero--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/design-system-collective-navbar--customize-ct-as (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/design-system-collective-navbar--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/design-system-styled-collective-card--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/design-system-styled-collective-card--frozen-collective (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-form--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--invoice (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--loading (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--receipt (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expenses-list--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expenses-list--states (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/host-dashboard-unhost-account-modal--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/host-dashboard-unhost-account-modal--with-balance (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/o-auth-application-approve-screen--default (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/virtual-cards-edit-virtual-card-modal--edit (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by kewitz** [comment]
Reverting this due to https://github.com/wojtekmaj/react-pdf/issues/1533

**Pull request merged**